src/lib/agent.c

```diff
@@ -44,15 +44,19 @@ DECLARE_SOF_UUID("sa", sa_uuid, 0x5276b491, 0x5b64, 0x464e,
 
 DECLARE_TR_CTX(sa_tr, SOF_UUID(sa_uuid), LOG_LEVEL_INFO);
 
-#define perf_sa_trace(pcd, sa)					\
```

Comment:
Using a macro preserve ```__func__``` and ```__LINE__```. Can you check the output is correct as before with this change ?